Source/Tools/LinuxEmulation/LinuxSyscalls/x32/Signals.cpp

```diff
@@ -239,6 +250,8 @@ void RegisterSignals(FEX::HLE::SyscallHandler* Handler) {
       siginfo_t* info64_p {};
 
       if (info) {
+        FaultSafeUserMemAccess::VerifyIsReadable(info, sizeof(*info));
+        info64 = *info;
```

Comment:
What's this added line doing here?

Reply:
This one was actually a bug, we were forgetting to copy over the data when provided

Reply:
Could you move functional changes to a different PR (or at least patch) for visibility? Hiding these in a NFC-sounding change is a bit surprising.

Reply:
Opened #4015 for the fix

Reply:
Rebased